src/components/navigation/navbar-links/NavBarLinks.tsx

```diff
@@ -0,0 +1,38 @@
+import './NavBarLinks.scss';
+
+import { ICustomer } from '../../../interfaces/customer.interface';
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+import { Role } from '../../../enums/role.enum';
+
+type Props = {
+  customer: ICustomer | undefined;
+};
+
+const NavBarLinks: React.FC<Props> = ({ customer }) => {
+  return (
+    <ul className="navbar-nav">
+      <li className="nav-item">
+        <NavLink to="/products" end className="nav-link">
+          <strong>Products</strong>
+        </NavLink>
+      </li>
+
+      <li className="nav-item">
+        <NavLink to="/cart" end className="nav-link">
+          <strong>My Cart</strong>
+        </NavLink>
+      </li>
+
+      {customer?.role === Role.ADMIN && (
```

Comment:
**suggestion:** consider the idea of having a `RoleGuard` component which avoid rendering its children based on role props it receives. This will avoid in the future to have complex if checks which might make code more unreadable.